src/cwe_checker_lib/src/analysis/forward_interprocedural_fixpoint.rs

```diff
@@ -283,3 +284,149 @@ pub fn create_computation_with_alternate_worklist_order<'a, T: Context<'a>>(
         default_value.map(NodeValue::Value),
     )
 }
+
+/// Generate a new computation from the corresponding context and an optional default value for nodes.
+/// Uses the Callee before Caller worklist order when computing the fixpoint.
+pub fn create_computation_with_callee_before_caller_worklist_order<'a, T: Context<'a>>(
+    problem: T,
+    default_value: Option<T::Value>,
+) -> super::fixpoint::Computation<GeneralizedContext<'a, T>> {
+    let mut graph = problem.get_graph().clone();
+    graph.retain_edges(|frozen, edge| !matches!(frozen[edge], Edge::Call(..)));
+    let priority_sorted_nodes: Vec<NodeIndex> = petgraph::algo::kosaraju_scc(&graph)
+        .into_iter()
+        .rev()
+        .flatten()
+        .collect();
+
+    let generalized_problem = GeneralizedContext::new(problem);
+    super::fixpoint::Computation::from_node_priority_list(
+        generalized_problem,
+        default_value.map(NodeValue::Value),
+        priority_sorted_nodes,
+    )
+}
+#[cfg(test)]
```

Comment:
Can you leave an empty line before `#[cfg(test)]`? I am surprised that `cargo fmt` apparently has no opinion about this...